test/e2e/fixtures/e2e_suite.go

```diff
@@ -187,6 +188,17 @@ func (s *E2ESuite) DeleteResources() {
 			err := archive.DeleteWorkflow(string(w.UID))
 			s.CheckError(err)
 		}
+		parse, err = labels.ParseToRequirements(Backfill)
+		s.CheckError(err)
+		backfillWorkflows, err := archive.ListWorkflows(utils.ListOptions{
+			Namespace:         Namespace,
+			LabelRequirements: parse,
+		})
+		s.CheckError(err)
```

Comment:
should we check backfill workflow actually exist, incase this returns empty list